README.md

```diff
@@ -1,42 +1,14 @@
-# Foxglove Studio WebSocket protocol libraries
+# Foxglove WebSocket Protocol
 
-This repository provides a protocol specification and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
+This repository provides a [protocol specification](docs/spec.md) and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
 
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The protocol is encoding-agnostic, and supports any data format as long as the desired encoding is supported by both client and server.
```

Comment:
I wonder if we should give a short overview of high-level protocol concepts here. Something like:

```suggestion
A Foxglove WebSocket server can provide multiple data streams, called _channels_. When a client subscribes to a channel, it begins receiving _messages_ on that channel. This protocol does not prescribe the format of message data — rather, the server specifies the _encoding_ of each channel, and the client uses this information to determine whether it can decode the messages. For information about which encodings are supported by Foxglove Studio, visit the [Studio documentation](https://foxglove.dev/docs/connection/foxglove-websocket).
```

Reply:
Revised slightly and added.